src/pages/backoffice/sor-contracts.js

```diff
@@ -0,0 +1,32 @@
+import {
+  AGENT_ROLE,
+  CONTRACTOR_ROLE,
+  CONTRACT_MANAGER_ROLE,
+  AUTHORISATION_MANAGER_ROLE,
+  OPERATIVE_ROLE,
+} from '@/utils/user'
+import Meta from '../../components/Meta'
+
+import { getQueryProps } from '../../utils/helpers/serverSideProps'
+import SORContracts from '../../components/BackOffice/SorContracts'
+
+const SORContractsPage = () => {
+  return (
+    <>
+      <Meta title="BackOffice" />
+      <SORContracts />
+    </>
+  )
+}
+
+export const getServerSideProps = getQueryProps
+
+SORContractsPage.permittedRoles = [
+  AGENT_ROLE,
+  CONTRACTOR_ROLE,
+  CONTRACT_MANAGER_ROLE,
```

Comment:
Could these be extracted to a shared file? As we're probably going to have the same access level for all parts of the BO

Reply:
the roles. just seeing a lot of repetition